docs/user-guide/downward-api/index.md

```diff
@@ -46,6 +52,20 @@ The `fieldRef` is evaluated and the resulting value is used as the value for
 the environment variable.  This allows users to publish their pod's name in any
 environment variable they want.
 
+The `resourceFieldRef` is a structure that has a `containerName` field, a `resource`
+field, and a `divisor` field. The `containerName` is the name of a container,
+whose resource (cpu or memory) information is to be exposed. The `containerName` is
+optional for environment variables and defaults to the current container. The
+`resource` field is an expression designating a resource in a container, and
+possible values are `limits.cpu`, `limits.memory`, `requests.cpu`, and `requests.memory`.
+The `divisor` field specifies an output format of the resource being exposed. The
+`divisor`'s possible values for cpu are `1`(cores)  and `1m`(millicores). The `divisor`'s
+possible values for memory in fixed point integer (decimal) are `1`(bytes), `1k`(kilobytes),
+`1M`(megabytes), `1G`(gigabytes), `1T`(terabytes), `1P`(petabytes), `1E`(exabytes),
+and in their power-of-two equivalents are `1Ki`(kibibyte), `1Mi`(mebibyte),
+`1Gi`(gibibyte), `1Ti`(tebibyte), `1Pi`(pebibyte), `1Ei`(exbibyte). If the `divisor`
+is not specified, it defaults to "1" for cpu and memory.
+
```

Comment:
What about to put the divisors into table? Something like [1]. The same for possible values (or make a list of possible values). So it is easier to get a picture of possible values from scratch.

[1] https://github.com/openshift/openshift-docs/blob/master/admin_guide/garbage_collection.adoc#image-garbage-collection